sale_procurement_group_by_requested_date/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Group procurements by requested date",
+    "summary": "Groups pickings based on requested date of order line",
+    "version": "9.0.0.0.1",
+    "category": "Sales Management",
+    "website": "http://www.eficent.com",
+    "author": "Eficent Business and IT Consulting Services S.L., "
```

Comment:
For the headers it's fine but this is shown in the form view of apps in odoo, and due to large length of our company name sometimes OCA is not shown properly and we really want to show OCA as author in the module description.